src/main/java/com/jherrick/BasicBoardEvaluator.java

```diff
@@ -1,28 +1,90 @@
 package com.jherrick;
 
 public class BasicBoardEvaluator implements BoardEvaluator{
+    
+    // =============== Instance variables ===============
+    
+    // Evaluation metrics. Only add or remove metrics, do not hard code in. Should be hard coded in EVAL_CONSTANTS
+    private static int c0 = Constants.EVAL_CONSTANTS[0];
+	private static int c1 = Constants.EVAL_CONSTANTS[1];
+	// private static int c2 = Constants.EVAL_CONSTANTS[2];
+
+    public BigBoard board;
+    public Side side;
+
+
+
+    // ============== Constructors ======================
+    public BasicBoardEvaluator(){
+        board = new BigBoard();
+        side = Side.X;
+    }
+    
+    public BasicBoardEvaluator(BigBoard b, Side s){
+        this();
+
+        board = b;
+        side = s;
+    }
+
+
+
+    // ============== Public Functions =================
     @Override
-    public double evaluate(Board board, Side side) {
+    public double evaluate(BigBoard board, Side side) {
```

Comment:
Same comment as above - as much as we can, all dependencies should be on Board. If it can't be on Board, we need to do some work on our interfaces.